src/lib/header-bidding/prebid/appnexus.ts

```diff
@@ -46,7 +47,7 @@ const getAppNexusDirectPlacementId = (
 	const defaultPlacementId = '9251752';
 	switch (getBreakpointKey()) {
 		case 'D':
-			if (isInFrontsBannerVariant) {
+			if (isInFrontsBannerVariant || includeBillboardsInMerchHigh()) {
```

Comment:
Could you update the comment to indicate that the merchandising-high slot doesn't include leaderboard sized ads either

Reply:
Great spot, will do!